mpact/services.py

```diff
@@ -284,8 +293,8 @@ def is_blank(val):
             days = row["Days"]
             message = row["Message"]
             comment = row["Comment"] or ''
-            if is_blank(days) or is_blank(message):
-                bad_rows[sheet["title"]].append(n)
+            if is_blank(days) or is_blank(message) or not is_positive_numeric(days):
+                bad_rows[sheet["title"]].append(n+1)
```

Comment:
Are you sure this is `n + 1`? `enumerate()` on line 292 is starting at 1 already. Should it start at 2?

Reply:
I probably should have added a comment there. 

The `enumerate()` starts at 1, which means it skips the 'first' row, which is the header. The `n+1` here is for the UI response to tell the user which row is incorrect. Excel / humans don't dp 0-indexed arrays, so added one to make it more clear which row has the problem.